icubam/www/handlers/home.py

```diff
@@ -40,6 +53,16 @@ def initialize(self, config, db_factory):
     super().initialize(config, db_factory)
     self.regions = None
 
+  def get_disclaimer_url(self):
+    if self.config.server.has_key('disclaimer'):
+      path = self.config.server.disclaimer
+      # The user has already agreed, we skip.
+      if path is not None and os.path.exists(path):
+        return "<a href='{}disclaimer'>disclaimer</a>".format(
+          self.config.server.base_url
```

Comment:
I don't think it should contain the `base_url`. For instance I start the dev server on a remote url for testing, and overall . This however redirects to localhost (for a dev server) and fails to load.